packages/extension/src/chromeServices/background/utils/focusOrCreatePopupWindow.ts

```diff
@@ -46,8 +48,8 @@ export const focusOrCreatePopupWindow = async ({
   requestMessage,
   receivingMessage,
   errorPayload,
-  width = NOTIFICATION_WIDTH,
-  height = NOTIFICATION_HEIGHT
+  width = isWindows ? NOTIFICATION_WIDTH + 16 : NOTIFICATION_WIDTH,
```

Comment:
Where does come from the `+16`?
Maybe we could import it from somewhere, it's probably a padding or something.